.github/ISSUE_TEMPLATE/Issue.yml

```
@@ -0,0 +1,97 @@
+name: 🐛 Issue Report
+description: Create an Issue to help us improve.
+title: "[🐛 Bug]: <title>"
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this New Issue! First, let's get some information about the environment you are using:
+
+  - type: input
+    id: Alumnium-version
+    attributes:
+      label: Alumnium Version
+      description: What version of Alumnium are you running?
+      placeholder: latest(0.8.0)
```

The latest version will be changing with time, let's not specify it in the placeholder
```suggestion
      placeholder: latest
```

---

will remove it from Placeholder